### ISSUE TITLE
Phase 1: Pretrain the causal Player representation from training-only supervision

### ISSUE THREAD

**Issue by rhythmatician** [rhythmatician/FractalSync#131]
Part of #130 and #77.

Coordinates with #104 (causal Player trunk/state contract), #108 (PlayerObservation), #109 (AudioActivation), #102 (SongFormer structural distillation), #103 (affect teachers), #95 (ColorIntent weak targets where retained), and #111 (diagnostics/evaluation).

- **Phase:** 1 — representation pretraining
- **Purpose:** teach the shared causal Player state `h_t` to encode useful musical/contextual information before learning Controls
- **Product Controls:** absent, frozen, or ignored during this phase
- **Behavioral reward / RL:** out of scope
- **Blocks:** the pretrained-initialization arm of Phase 2; does **not** block Phase-2 RL from random initialization

## Goal

Build and validate a strictly causal representation-pretraining path:

```text
PlayerObservation history O_{<=t}
        -> shared causal trunk f_theta
        -> h_t
           |-- boundary-now head
           |-- transition-readiness head(s)
           |-- section/function head
           |-- valence/arousal head(s)
           `-- other evidence-gated training-only heads
        -> direct training targets
        -> supervised / self-supervised representation losses
```

This phase answers:

> **What should the Player's causal hidden state understand about what is happening in the music?**

It does **not** answer what steering, throttle, grip, impulse, Julia zoom, rotation, or ColorIntent action should be taken.

## Core invariant

Phase 1 has no requirement for a correct Controls target and no requirement for behavioral reward.

Offline/noncausal teachers may use whole-song or future context **only on the target/loss side**. The Student input/state remains strictly causal:

```text
h_t = f_theta(O_t, h_{t-1})
```

Changing future audio may change a teacher target at `t`; it must not change the causal Student input or hidden state already constructed at `t`.

## Inputs / authority

Consume rather than redefine:

- #109 `AudioActivation/1`;
- #108 versioned `PlayerObservation`;
- #104 selected causal trunk and hidden-state semantics;
- canonical sample-clock timing;
- ADR 0001/#119 Rust-owned deterministic runtime semantics.

Phase-1 observation sequences may come from deterministic scripted/manual/baseline trajectories, frozen datasets, or another explicit versioned source, but they must preserve destination observation semantics and exact chronology.

Do not create a trainer-only feature layout.

## Training targets

Initial target families are supplied by separate research tickets rather than invented here:

### #102 SongFormer structure

Candidate training-only objectives include:

```text
L_B  boundary-now distillation
L_R  future-horizon transition readiness
L_F  section/function distribution distillation
```

### #103 affect

Candidate objectives include confidence-weighted valence/arousal losses:

```text
L_VA
```

### #95 weak ColorIntent target

May participate only if #95/#103 evidence retains it as representation supervision.

### Other heads

Any later auxiliary/self-supervised target must have a concrete downstream hypothesis and a keep/drop ablation. Do not accumulate heads because targets are easy to generate.

Conceptually:

```text
L_repr
  = lambda_B  L_B
  + lambda_R  L_R
  + lambda_F  L_F
  + lambda_VA L_VA
  + ...
```

The exact retained composition is empirical.

## Training mechanics

Use ordinary gradient-based backpropagation through the model trunk and auxiliary heads.

No gradient through Mandelbrot Physics, Controls interpretation, Julia presentation integration, or renderer is required.

The product Controls head may be:

- not instantiated;
- instantiated but frozen;
- instantiated but excluded from the loss.

Whichever path is selected must make the absence of Controls supervision explicit.

## Hidden-state chronology / batching

Consume #104 semantics for:

- stream/excerpt start;
- hidden-state initialization;
- sequence/chunk boundaries;
- truncated BPTT where used;
- song/performance epoch changes;
- checkpoint save/load.

A gradient truncation boundary may stop backward flow. It may not silently reset forward hidden state unless that reset is part of the explicit experiment.

## Teacher alignment and leakage

Provide a single loss-side target interface with deterministic alignment to canonical Player ticks.

Verify mechanically:

- future-aware teacher outputs never enter `AudioActivation` or `PlayerObservation`;
- teacher window padding/end-of-song behavior cannot identify targets through Student input artifacts;
- normalization/calibration is fit only on training folds where applicable;
- overlapping windows from one performance do not cross grouped train/held-out boundaries;
- teacher model/version/time-alignment metadata is stamped with the experiment.

## Checkpoint handoff contract

The primary product of this issue is a reusable pretrained trunk checkpoint compatible with Phase 2.

Stamp at least:

- AudioActivation version;
- PlayerObservation version;
- PlayerPolicy/trunk architecture version;
- hidden-state contract version;
- teacher target/artifact versions;
- training-data split identity;
- training configuration / seed;
- normalization/calibration versions.

Auxiliary heads may be discarded after training. Their removal must not change product input/output schemas.

## What success means

Auxiliary accuracy is diagnostic, not sufficient acceptance.

The decisive test is downstream transfer in the Phase-2 ticket:

```text
RL from random initialization
vs
RL from this Phase-1 pretrained trunk
```

Measure whether pretraining improves some combination of:

- RL sample efficiency;
- training stability;
- transition preparation/timing;
- held-out musical behavior;
- eyes-on coherence;
- robustness across songs and reset states.

A head that predicts its teacher well but does not contribute to downstream improvement should be dropped.

## Required tests

At minimum:

1. **Causality:** future audio cannot alter already-constructed Student inputs/state.
2. **Teacher isolation:** future-aware targets remain loss-side only.
3. **Controls independence:** Phase 1 trains with no Controls target and no behavioral reward.
4. **Sequence chronology:** chunking preserves forward causal state subject to explicit reset semantics.
5. **BPTT semantics:** gradient truncation does not silently reset hidden state.
6. **Grouped split leakage:** windows from one performance cannot leak across train/test folds.
7. **Checkpoint round-trip:** saved trunk state reloads with identical contract metadata and deterministic inference.
8. **Head removal:** auxiliary heads can be removed without changing the product observation/Controls schemas.

## Acceptance

- [ ] A strictly causal Phase-1 sequence/training path exists over destination PlayerObservation semantics.
- [ ] At least one real training-only teacher family is attached through the generic auxiliary-target seam.
- [ ] No Controls target or behavioral reward is required.
- [ ] No future/offline teacher value enters Student runtime inputs.
- [ ] Hidden-state reset/chunk/BPTT semantics match #104.
- [ ] Teacher alignment, provenance, normalization, and grouped splits are reproducible.
- [ ] A versioned pretrained trunk checkpoint can initialize the Phase-2 Player without schema adaptation.
- [ ] Auxiliary heads are removable at export.
- [ ] Each retained head has an explicit keep/drop result based on downstream Phase-2 behavior, not auxiliary prediction alone.
- [ ] At least one matched Phase-2 comparison reports random-init vs pretrained-init behavior/sample efficiency.

## Non-goals

- Do not learn Controls v2 here.
- Do not optimize behavioral reward here.
- Do not run PPO/actor-critic or other policy optimization here.
- Do not differentiate through Physics or renderer.
- Do not promote teacher outputs into runtime AudioActivation/PlayerObservation.
- Do not make symbolic section/emotion labels part of the product API.
- Do not redesign PlayerPolicy architecture, Controls, Physics, or observation contracts here.
